.github/workflows/aks.yaml

```diff
@@ -26,6 +26,13 @@ env:
 jobs:
   check_changes:
     name: Deduce required tests from code changes
+    if: |
+      (github.event.issue.pull_request && (
+        startsWith(github.event.comment.body, 'ci-aks') ||
+        (startsWith(github.event.comment.body, 'test-me-please') && (needs.check_changes.outputs.tested == 'true'))
```

Comment:
I think `needs.check_changes.outputs.tested == 'true'` should be left in `installation-and-connectivity.if`, no? Otherwise, how can a job reference its own outputs?

Reply:
:facepalm: Thanks Nicolas!

Reply:
I simply duplicated the conditions instead. We could probably optimize this later to avoid duplication (with a new definition of `tested:`), but that would require more tests. Probably best to fix the concurrency groups now and optimize later.

Reply:
I'm not sure I follow, can't we have this simple `if` condition?

```yaml
if: needs.check_changes.outputs.tested == 'true'
```

Reply:
No because `needs.check_changes.outputs.tested` isn't defined for `scheduled` workflow runs. I also on purpose allowed users to bypass the `paths-filter` check when using the `ci-xxx` trigger phrases (though that probably matters less).

Reply:
Ah, I think I got it. Indeed untangling the `if` conditions is not so simple. Let's fix later.